### PR TITLE
read default schema from a file

### PR DESCRIPTION
**Pull request by krunalsshah**
read default schema from a file

### CODE REVIEW

File: ern-api-gen/src/generateProject.ts [hunk 1]
```diff
@@ -105,112 +106,7 @@ export async function generateInitialSchema({
...
-        },
-        "desc": {
-          "type": "string"
-        }
-      }
-    }
-  }
-}
-`
+    : fileUtils.readFile(path.join(__dirname, '..', INITIAL_SCHEMA_FILE))
```
Comment (friederbluemle) [inline ern-api-gen/src/generateProject.ts]: Both paths of the conditional are now `fileUtils.readFile(..)`. Can this be simplified?

File: ern-api-gen/initialApiSchema.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        "name": {
+          "type": "string"
+        },
+        "desc": {
+          "type": "string"
+        }
+      }
+    }
+  }
+}
```
Comment (friederbluemle) [inline ern-api-gen/initialApiSchema.json]: Missing newline at EOF.

Comment (krunalsshah) [reply ern-api-gen/initialApiSchema.json]: I remember the atom editor adds the EOF automatically. Looks will have to set the option to do so for Visual Studio Code.

File: ern-api-gen/src/generateProject.ts (hunk 1 above)
Comment (krunalsshah) [reply ern-api-gen/src/generateProject.ts]: 👍 modified the conditional

**Pull request merged**